server/config.ts

```diff
@@ -122,6 +118,7 @@ export default {
     },
     coordinatorApi: {
       url: get('COORDINATOR_API_URL', 'https://arns-coordinator-api-dev.hmpps.service.justice.gov.uk'),
+      healthPath: '/health/ping',
```

Comment:
Apologies @alexw-moj my GitHub just messed up and deleted your comment. For future reference, it was a question along the lines of "I've disabled all the containers but this health check continued to pass".

My follow up comment was; did you ensure to stop the Wiremock container too? On our local development environment, it uses the wiremock for the coordinator.